tequilapi/client/client.go

```diff
@@ -239,3 +239,16 @@ func (client *Client) Stop() error {
 
 	return nil
 }
+
+// GetSessions returns all sessions from history
+func (client *Client) GetSessions() (endpoints.SessionsDTO, error) {
```

Comment:
`sessions := endpoints.SessionsDTO{}`
and using only `sessions` should be simpler.

Reply:
refactored